packages/core/src/domain/patient-settings.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseDomain, BaseDomainCreate } from "./base-domain";
+
+export interface PatientSettingsCreate extends BaseDomainCreate {
+  id: string;
```

Comment:
Could move this to `PatientSettings` so the "create" represents the input to the create command function.

Reply:
I don't see it on the PR, maybe forgot to push the commit?

Reply:
Actually, we don't need this here as the `id` is already part of the `BaseDomainCreate`.